src/reactviews/pages/ExecutionPlan/executionPlanGraph.tsx

```diff
@@ -3,180 +3,330 @@
+    const state = useContext(ExecutionPlanContext);
+    const executionPlanState = state?.state;
+    const [isExecutionPlanLoaded, setIsExecutionPlanLoaded] = useState(false);
+    const [query, setQuery] = useState("");
+    const [cost, setCost] = useState(0);
+    const [executionPlanView, setExecutionPlanView] =
+        useState<ExecutionPlanView | null>(null);
+    const [zoomNumber, setZoomNumber] = useState(100);
+    const [customZoomClicked, setCustomZoomClicked] = useState(false);
+    const [findNodeClicked, setFindNodeClicked] = useState(false);
+    const [findNodeOptions, setFindNodeOptions] = useState<string[]>([]);
+    const [highlightOpsClicked, setHighlightOpsClicked] = useState(false);
+    const [propertiesClicked, setPropertiesClicked] = useState(false);
+    const [propertiesWidth, setPropertiesWidth] = useState(400);
+    const [containerHeight, setContainerHeight] = useState("100%");
+    const resizableRef = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+        if (!executionPlanState || isExecutionPlanLoaded) return;
+
+        setContainerHeight(
+            executionPlanState!.executionPlanGraphs!.length > 1
+                ? "500px"
+                : "100%",
+        );
+
+        // @ts-ignore
+        window["mxLoadResources"] = false;
+        // @ts-ignore
+        window["mxForceIncludes"] = false;
+        // @ts-ignore
+        window["mxResourceExtension"] = ".txt";
+        // @ts-ignore
+        window["mxLoadStylesheets"] = false;
+        // @ts-ignore
+        window["mxBasePath"] = "./src/reactviews/pages/ExecutionPlan/mxgraph";
+
+        const mxClient = azdataGraph.default();
+
+        function loadExecutionPlan() {
+            if (executionPlanState && executionPlanState.executionPlanGraphs) {
+                const executionPlanRootNode =
+                    executionPlanState.executionPlanGraphs[graphIndex].root;
+                const executionPlanView = new ExecutionPlanView(
+                    executionPlanRootNode,
+                );
+                const executionPlanGraph = executionPlanView.populate(
+                    executionPlanRootNode,
+                );
+
+                const div = document.getElementById(
+                    `queryPlanParent${graphIndex + 1}`,
+                );
+                // create a div to hold the graph
+                const queryPlanConfiguration = {
+                    container: div,
+                    queryPlanGraph: executionPlanGraph,
+                    iconPaths: utils.getIconPaths(),
+                    badgeIconPaths: utils.getBadgePaths(),
+                    expandCollapsePaths: utils.getCollapseExpandPaths(),
+                    showTooltipOnClick: true,
+                };
+                const pen = new mxClient.azdataQueryPlan(
+                    queryPlanConfiguration,
+                );
+                pen.setTextFontColor("var(--vscode-editor-foreground)"); // set text color

```

Comment:
NIT: save this in a const variable and use it everywhere.

Reply:
I think everywhere else automatically uses this color for text, so that's why we only need to set it here, for the query plan graph